package/samplers/smac_sampler/sampler.py

```diff
@@ -99,6 +99,7 @@ def __init__(
         self,
         search_space: dict[str, BaseDistribution],
         n_trials: int = 100,
+        seed: int = -1,
```

Comment:
if `seed` is set as `-1`, SMAC will generate a random seed and we cannot guarantee the reproducibility. This argument here is not so intuitive to users, so it would be best if we can also mark this in the docs (such that users will change this to another value if they require reproducibility)